stock_no_negative/readme/CONTRIBUTORS.rst

```diff
@@ -1,6 +1,7 @@
 * Alexis de Lattre <alexis.delattre@akretion.com>
 * ForgeFlow S.L. <contact@forgeflow.com>
   * Jordi Ballester
+  * Joan Mateu
 * Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
 * Tecnativa <https://www.tecnativa.com>
   * Pedro M. Baeza
```

Comment:
This is formatting wrongly:

![image](https://user-images.githubusercontent.com/23449160/146921573-9847e5db-95d8-42c3-8275-5f67d2cc9aff.png)

Could you fix it?